index.html

```diff
@@ -1167,15 +1167,22 @@ <h3>Malicious Issuers and Verifiers</h3>
 tracking or sharing the presentation of particular credentials.
       </p>
       <p>
-A malicious [=issuer=] might intentionally attack group privacy by creating a
-unique status list per credential issued in order to establish a one-to-one mapping to track
-when a [=verifier=] processes a specific credential. Similarly, they could establish
-another a one-to-one mapping by using a different cryptographic key for every credential
-issued that is tracked in a status list.
+A malicious [=verifier=] might intentionally attack group privacy by sharing
+information from presented credentials with a malicious [=issuer=]. This
+sort of collusion is difficult to detect as it is typically performed in a
+secure communication channel between the [=issuer=] and the [=verifier=].
       </p>
       <p>
-A malicious [=verifier=] might intentionally attack group privacy by sharing
-information from presented credentials with a malicious [=issuer=].
+A malicious [=issuer=] might intentionally attack group privacy by creating a
+unique status list per credential issued in order to establish a one-to-one
+mapping to track when a [=verifier=] processes a specific credential. Similarly,
+they could establish another a one-to-one mapping by using a different
+cryptographic key for every credential issued that is tracked in a status list.
+This sort of collusion can be detected by [=holder=] software by detecting
+if the global identifiers used within a [=verifiable credential=] are shared
+by other credentials. [=Holders=] could then be warned when presenting a
+[=verifiable credential=] that contains global identifiers that are unique to
+that credential.
```

Comment:
Based on the conversation between @dlongley and @David-Chadwick...
```suggestion
This sort of collusion can be detected by [=holder=] software that serves
multiple [=holders=] (e.g., a [=holder=] app that runs on a server) if it
finds some global identifier(s) used within a [=verifiable credential=]
that are shared by other credentials. [=Holders=] could then be warned when
presenting a [=verifiable credential=] that contains some global identifier(s)
that are unique to that credential.
```

Reply:
Thanks @TallTed this is a much better explanation. But it begs the privacy issue of a holder server that is scanning multiple users' credentials.

Reply:
Try this, @David-Chadwick (and @dlongley, @msporny, others)
```suggestion
This sort of collusion can be detected by [=holder=] software that serves
multiple [=holders=] (e.g., a [=holder=] app that runs on a server) if it
has, for example, an opt-in process finds some global identifier(s) used
within a [=verifiable credential=] that are shared by other credentials.
[=Holders=] could then be warned when presenting a [=verifiable credential=]
that contains some global identifier(s) that are unique to that credential.
Such an opt-in service could represent some additional privacy concerns;
whether this potential exposure via the [=holder=] software is justified by
the awareness of possible global identifier correlation can only be evaluated
by the users of such a system.
```

Reply:
minor tweak

Reply:
Change to
> has, for example, an opt-in process that finds some global identifier(s) used

Change to
> within a [=verifiable credential=]  are shared by other credentials.

Reply:
> that contains some global identifier(s) that are unique to that credential.

they cannot be unique if they are shared. So suggest

> that contains some global identifier(s) that are shared with other credential(s).

Reply:
Modified based on [this comment from @David-Chadwick](https://github.com/w3c/vc-bitstring-status-list/pull/161/files#r1548325764)... 

Not modified based on [this comment from @David-Chadwick](https://github.com/w3c/vc-bitstring-status-list/pull/161/files#r1548331079), based on my understanding of the process suggested by @dlongley. @dlongley — Please help correct, if I've got it wrong!
```suggestion
This sort of collusion can be detected by [=holder=] software that serves
multiple [=holders=] (e.g., a [=holder=] app that runs on a server) if it
has, for example, an opt-in process that finds that some global identifier(s) 
used within a [=verifiable credential=] are shared by other credentials.
[=Holders=] could then be warned when presenting a [=verifiable credential=]
that contains some global identifier(s) that are unique to that credential.
Such an opt-in service could represent some additional privacy concerns;
whether this potential exposure via the [=holder=] software is justified by
the awareness of possible global identifier correlation can only be evaluated
by the users of such a system.
```

Reply:
If the (globally unambiguous) identifiers are shared amongst many different VCs, the privacy threat is actually eliminated. It's the case where an individual VC can be unambiguously identified in an unexpected way that is the problem.

Reply:
I agree, but this paragraph is talking about IDs that are shared, not about IDs that are not shared. If you want to say something about IDs that are not shared it should be in a new paragraph.